client/src/components/Carousel/Carousel.js

```diff
@@ -0,0 +1,126 @@
+import React, { useState } from "react";
+import styles from "../Carousel/Carousel.module.css";
+import "react-responsive-carousel/lib/styles/carousel.min.css";
+import { Carousel } from "react-responsive-carousel";
+import TaskStatusChart from "../TaskStatusChart/TaskStatusChart";
+import PropTypes from "prop-types";
+
+export const InsightsCarousel = ({ taskData }) => {
+  const startedTasks = {
+    early: [],
+    onTime: [],
+    late: [],
+  };
+
+  const EndedTasks = {
+    early: [],
+    onTime: [],
+    late: [],
+  };
+
+  const summaryTasks = {
+    completed: [],
+    onGoing: [],
+    notStarted: [],
+  };
+
+  const manageStartedTasks = (
+    sampleTaskData,
+    startedTasks,
+    EndedTasks,
+    summaryTasks
+  ) => {
+    const totalTasks = sampleTaskData.length;
+
+    for (const t of sampleTaskData) {
```

Comment:
I would just make a small note that we're comparing time by minutes

Reply:
I would rename `t` to `task` or something more appropriately

Reply:
Resolved: I made this change.